fileattrs/elf.attr

```diff
@@ -1,5 +1,6 @@
-%__elf_provides		%{_rpmconfigdir}/elfdeps --provides --multifile
-%__elf_requires		%{_rpmconfigdir}/elfdeps --requires --multifile
+%__elf_depstyle		%[ %_rpmformat >=6 ? "--multiarch" : "--biarch" ]
+%__elf_provides		%{_rpmconfigdir}/elfdeps --provides --multifile %{__elf_depstyle}
+%__elf_requires		%{_rpmconfigdir}/elfdeps --requires --multifile %{__elf_depstyle}
```

Comment:
We would want to generate biarch and multiarch provides with rpm v6 files, right? And only switch between biarch or multiarch requires. Otherwise the mix of v4 and v6 would be broken.

Reply:
Generating v4 compatibility provides would be up to he distributor. We're not going to pollute the v6 space with that crud by default :smile: